packages/safety-suite-api/src/api/dashboardSink/riskScores.ts

```diff
@@ -0,0 +1,58 @@
+import {Request} from '@idelic/safety-net';
+import {ApiOptions, ApiSuccessResponse, runApi} from 'src';
+
+import {BaseFields, Query, TableQuery} from './types';
+import {convertSortsToStrings} from './util';
+
+export type RiskScoreBuckets =
+  | '1'
+  | '11'
+  | '21'
+  | '31'
+  | '41'
+  | '51'
+  | '61'
+  | '71'
+  | '81'
+  | '91';
+
+export type RiskScoreGraph = {[R in RiskScoreBuckets]: number};
```

Comment:
Using a mapped type here works fine, another way to achieve this is by using the global `Record` type. The equivalent to the above would be:
```ts
export type RiskScoreGraph = Record<RiskScoreBuckets, number>;
```